core/services/keeper/registry_synchronizer_log_listener.go

```diff
@@ -28,10 +28,13 @@ func (rs *RegistrySynchronizer) IsV2Job() bool {
 func (rs *RegistrySynchronizer) HandleLog(broadcast log.Broadcast) {
 	log := broadcast.DecodedLog()
 	if log == nil || reflect.ValueOf(log).IsNil() {
-		logger.Error("HandleLog: ignoring nil value")
+		logger.Error("RegistrySynchronizer: HandleLog: ignoring nil value")
 		return
 	}
 
+	txHash := broadcast.RawLog().TxHash.Hex()
+	logger.Debugf("RegistrySynchronizer: received log, waiting for confirmations, jobID: %d, type: %T, txHash: %s", rs.job.ID, log, txHash)
```

Comment:
Personally I prefer `Debugw` it uses structured logging to print and is easier to read, e.g:

```Debugw("RegistrySynchronizer: received log, waiting for confirmations", "jobID", rs.job.ID...```